Classes/Neos/Neos/Ui/Domain/Model/Changes/CopyInto.php

```diff
@@ -11,8 +12,57 @@
  * source code.
  */
 
+use Neos\ContentRepository\Domain\Model\NodeInterface;
+
 class CopyInto extends AbstractCopy
 {
+
+
+    /**
+     * @var string
+     */
+    protected $parentContextPath;
```

Comment:
So this is in fact a target parent node, right? Maybe call it smh like `referenceNodeContextPath` or `targetNodeContextPath`?